### ISSUE TITLE
Error when upgrading with package for PHP7

### ISSUE THREAD

**Issue by lonesomewalker** [chamilo/chamilo-lms#2359]
### Current behavior / Resultado actual / Résultat actuel
When upgrading 1.10 to 1.11.6, you get an:
`Uncaught TypeError: Return value of Doctrine\Common\Annotations\AnnotationRegistry::registerFile() must be an instance of Doctrine\Common\Annotations\void`

### Expected behavior / Resultado esperado / Résultat attendu
Package should be modified to meet the MINIMUM requirements (PHP 7.0!).

### Chamilo Version / Versión de Chamilo / Version de Chamilo
1.11.6

Fix for me was to explicit set 
```
    "config": {
        "platform": {
            "php": "7.0"
        },
```
in composer.json and after resolving with composer update it noticed that another git file is missing -> download -> everything went smooth again :-)

Also, if this package is made for PHP7, it would be possible to change:
```
  "require": {
        "php": ">=5.5",
```
to the according version ;-)

#2347

**Comment by ywarnier** [comment]
Hi @lonesomewalker,
You don't say on which PHP version you ran that upgrade...

**Comment by lonesomewalker** [comment]
Debian latest and PHP 7.0
When running 7.1 no errors, 7.2 no errors.

**Comment by ywarnier** [comment]
You upgraded using the downloadable package (.zip or .tar.gz), or using git pull?

**Comment by lonesomewalker** [comment]
zip file

**Comment by ywarnier** [comment]
Yeah... this is indeed related to #2347 and is the reason why we haven't officially announced 1.11.6 to the public yet. This will be solved when closing #2347, so I'm closing this one as duplicate.
